src/constants.ts

```diff
@@ -7,3 +7,5 @@ export const CaCx_TREATMENT_ENCOUNTER_TYPE =
 export const MCH_EID_SUMMARY_ENCOUNTER_TYPE =
   "6d647bd4-33d6-4d04-a04a-595d2159b456";
 export const MCH_INTEGRATED_ANTENATAL_REGISTER_ENCOUNTER_TYPE = "";
+export const MCH_INTEGRATED_POSTNATAL_REGISTER_ENCOUNTER_TYPE =
```

Comment:
@akileng56 let's change the naming of these encounter types. 
Encounter types are named as simply Antenatal Encounter type or Postnatal Encounter Type.

Reply:
Updated